### DISCUSSION TITLE
Extending cockpit graphics

### DISCUSSION THREAD

**Discussion by binarte** [dxx-rebirth/dxx-rebirth#701] [Ideas]
I don't like having a distorted-looking cockpit playing at an aspect ratio wider than 4×3, being an artist I would be happy to produce graphics to extend the cockpit view beyond the existing cockpit image, would this be an welcome addition to the main branch?

**Comment by vLKp** [comment]
I expect most people turn off the cockpit view in favor of status-bar view or fullscreen view, both of which should handle non-4x3 aspects cleanly.  However, if you want to provide new cockpit assets under a license that permits royalty free distribution, I have no objection to it being included in the project.
